treeherder/webapp/api/jobs.py

```diff
@@ -134,19 +135,24 @@ def cancel(self, request, project, jm, pk=None):
         else:
             return Response("No job with id: {0}".format(pk), 404)
 
-    @detail_route(methods=['post'], permission_classes=[IsAuthenticated])
+    @list_route(methods=['post'], permission_classes=[IsAuthenticated])
     @with_jobs
-    def retrigger(self, request, project, jm, pk=None):
+    def retrigger(self, request, project, jm):
         """
         Issue a "retrigger" to the underlying build_system_type by scheduling a
         pulse message.
         """
-        job = jm.get_job(pk)
-        if job:
-            jm.retrigger(request.user.email, job[0])
-            return Response({"message": "retriggered job '{0}'".format(job[0]['job_guid'])})
+        if isinstance(request.data, QueryDict):
+            job_id_list = request.data.getlist("job_id_list")
         else:
-            return Response("No job with id: {0}".format(pk), 404)
+            job_id_list = request.data["job_id_list"]
```

Comment:
@maurodoglio I noticed there is a dissimilarity between post in Django and post from AngularJS.
request.data sent from AngularJS directly gets converted to a dictionary in Django and I can't use getlist:
```python
  File "/home/vagrant/venv/local/lib/python2.7/site-packages/rest_framework/views.py", line 453, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/vagrant/treeherder/treeherder/webapp/api/utils.py", line 200, in use_jobs_model
    return model_func(*args, jm=jm, **kwargs)
  File "/home/vagrant/treeherder/treeherder/webapp/api/jobs.py", line 145, in retrigger
    job_id_list = request.data.getlist("job_id_list")
AttributeError: 'dict' object has no attribute 'getlist'
```

Whereas post from Django ( in test_jobs_api.py ) is a QueryDict object, so I have to use a getlist() method there.